chapter_11_ansible.asciidoc

```diff
@@ -303,7 +322,7 @@ for.
 
     - name: Install docker  #<1>
       ansible.builtin.apt:  #<2>
-        name: docker  #<3>
+        name: docker.io  #<3>
```

Comment:
i did not have to do this - are you definitely on the same version of ubuntu?  22.04 jammy?

Reply:
for the record - my mistake, yes.